actors/runtime/src/main/java/cloud/orbit/actors/runtime/ActorProfiler.java

```diff
@@ -44,7 +43,7 @@
 {
     private final HashMap<Object, ProfilerData> collectedData = new HashMap<>();
     private final ReadWriteLock rwlock = new ReentrantReadWriteLock();
-    private final KryoCloner kryoCloner = new KryoCloner();
+    private final KryoSerializer kryoCloner = new KryoSerializer();
```

Comment:
Why isn't this a MessageSerializer or ObjectInstanceCloner for the field type so if we swap out the implementation we don't change the type?